index.html

```diff
@@ -0,0 +1,234 @@
+    <title>Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins&family=Roboto:wght@500&display=swap" rel="stylesheet">
+    <link rel="stylesheet" href="https://fonts.googleapis.com/css2?family=Material+Symbols+Outlined:opsz,wght,FILL,GRAD@48,400,0,0" />
+    <link rel="stylesheet" href="style.css">
+</head>
+<body>
+    <div id="container">
+        <div id="template1">
+            <header>
+                <a href="#" class="logo">My Logo</a>
+                <nav class="Union">
+                    <img src="images/header/Union.png" alt="bar">
+                </nav>
+            </header>
+            <div class="bg_image"></div>
+                <section id="landing_page">
+                    <div class="intro">
+                        <h1>I’m Praveen Glad to see you!</h1>
+                    </div>
+                    <div class="description">
+                        <p>I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.</p>
+                    </div>
+                    <div class="connect">
+                        <h3>LET'S CONNECT</h3>
+                    </div>
+                    <ul>
+                        <li><a href=""><img class="twitter" src="images/header//twiter.png" alt="twiter_logo"></a></li>
+                        <li><a href=""><img class="linkedIn" src="images/header//linkedin.png" alt="linkedIn_logo"></a></li>
+                        <li><a href=""><img class="m" src="images/header//m.png" alt="m_logo"></a></li>
+                        <li><a href=""><img class="github" src="images/header//github.png" alt="github_logo"></a></li>
+                        <li><a href=""><img class="ok" src="images/header//ok.png" alt="ok_logo"></a></li>
+                    </ul>
+                </section>
+        </div>
+         <!-- work -->
+         <div id="work_1">
+            <div class="a">
+                <img src="images/header/tonic.png" alt="tonic">
+            </div>
+            <div class="b">Tonic</div>
+            <div class="c">
+                <div class="canopy">CANOPY</div>
+                <div class="dot">•</div>
+                <div class="job">Back End Dev</div>
+                <div class="dot">•</div>
+                <div class="yrs">2015</div>
+            </div>
+            <div class="d">
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required.</p>
+            </div>
+            <div class="e">
+                <button class="one">html</button>
+                <button class="two">css</button>
+                <button class="three">javaScript</button>
+            </div>
```

Comment:
- [ ] Kindly update this part to make it a `ul` with `li` tags, each one of them contains a technology 👍 
Hint:
```
<ul>
    <li>HTML</li>
    <li>CSS</li>
    <li>JavaScript</li>
</ul>
```